src/data/sortOption.ts

```diff
@@ -0,0 +1,5 @@
+export const SORT_OPTIONS: SortOption[] = [
+    'Most popular',
+    'Price: lowest to highest',
+    'Price: highest to lowest',
+];
```

Comment:
🟢 good thinking to have the sort options in a separate file, integrating with the backend should be much easier with this choice